### ISSUE TITLE
Bug in LlamaMLP layer to detect neurons (inside Neuron Detection folder)

### ISSUE THREAD

**Issue by RadiantCrystal** [DAMO-NLP-SG/multilingual_analysis#6]
Is not it bug in the given line https://github.com/DAMO-NLP-SG/multilingual_analysis/blob/main/neuron_detection/transformers/models/llama/modeling_llama.py#L246

`return real_output, torch.sum(torch.abs(self.up_proj(x)), dim=1).squeeze().tolist(), torch.sum(torch.abs(self.up_proj(x)), dim=1).squeeze().tolist()`

Should it not be self.up_proj(x) and self.down_proj(x)?

Can authors please clarify this?

**Comment by yyjiao97** [comment]
The same question, hoping the author can answer, otherwise cannot believe in the authenticity of the paper.

**Comment by Zongmin741** [comment]
Hey, if you look at the proof in the paper, the importance of neurons of up_proj and down_proj is calculated the same way.
